concepts/05 UI Components/Popup/00 Getting Started with Popup/08 Configure User Interaction.md

```diff
@@ -0,0 +1,98 @@
+Users can drag the popup. To configure this functionalty, use the [dragEnabled](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#dragEnabled) property.
```

Comment:
The descriptions follow the same pattern over and over again, it seems robotic. Let's change it a bit:

```suggestion
The Popup supports drag and drop. To enable this functionalty, set the [dragEnabled](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#dragEnabled) property to **true**. Users drag the Popup by its title bar, so make sure that you [configured it]().
```